### DISCUSSION TITLE
Welcome to Seeed-LoRa-E5 Discussions!

### DISCUSSION THREAD

**Discussion by danak6jq** [danak6jq/Seeed-LoRa-E5#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LucasAMPA** [comment]
Hello, my name is Lucas! I am an Electrical Engineer and was trying this repo out on my LoRa-E5 device.

Whenever I try to do OTAA join requests my device isn't able to join. However this repo looks to be one of the better maintained ones for this board. The info on the wiki page tutorial is getting dated. Wondering if you had any recommendations for the TTN side.

More technical details about my question are here: https://www.thethingsnetwork.org/forum/

**Reply by danak6jq** [reply]
Specifically which forum post at TTN? Still having trouble?

**Reply by LucasAMPA** [reply]
Apologies, it's this forum post here: https://www.thethingsnetwork.org/forum/t/wio-e5-dev-board-otaa-join-failed/64213/7

Yes, I am still having trouble where I am unable to see any traffic from my end node on the gateway. I am assuming you should just be able to download the repo, flash, and at least see the OTAA attempts? The blue TX light is indeed flashing at least.

**Reply by danak6jq** [reply]
What band is your GW set for?

**Reply by LucasAMPA** [reply]
@danak6jq I figured out my issue:

The way the repo is written it defaults to FSB 1. So when I was going into TTI and using FSB2 nothing was able to be seen. Additionally, I realized that over-writing the factory bin file from wio-E5 makes it impossible to get the actual devEui and AppEui make the device useless. Thank you for this very nice repo, it has helped me a lot.

**Comment by LucasAMPA** [comment]
US 915 Mhz

On Mon, Aug 7, 2023 at 7:46 PM Dana Myers ***@***.***> wrote:

> What band is your GW set for?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/danak6jq/Seeed-LoRa-E5/discussions/18#discussioncomment-6663545>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7BFMTNYBGGS3KV5JG4XSW3XUF46XANCNFSM6AAAAAAS2VHKTA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 
This email message is for the sole use of the intended recipient(s) and may 
contain confidential and privileged information. Any unauthorized review, 
use, disclosure or distribution is prohibited. If you are not the intended 
recipient, please contact the sender by reply email and destroy all copies 
of the original message. If you are the intended recipient, please be 
advised that the content of this message is subject to access, review and 
disclosure by the sender's email administrator.

**Comment by LucasAMPA** [comment]
Sorry, to be more specific 902 - 923 TTN FSB 2

On Mon, Aug 7, 2023 at 7:46 PM Dana Myers ***@***.***> wrote:

> What band is your GW set for?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/danak6jq/Seeed-LoRa-E5/discussions/18#discussioncomment-6663545>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7BFMTNYBGGS3KV5JG4XSW3XUF46XANCNFSM6AAAAAAS2VHKTA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 
This email message is for the sole use of the intended recipient(s) and may 
contain confidential and privileged information. Any unauthorized review, 
use, disclosure or distribution is prohibited. If you are not the intended 
recipient, please contact the sender by reply email and destroy all copies 
of the original message. If you are the intended recipient, please be 
advised that the content of this message is subject to access, review and 
disclosure by the sender's email administrator.
